docs/UsingSpiffdemo/Basic_Tools.md

```diff
@@ -0,0 +1,228 @@
+# Building Blocks of the System
+These process models serve as the foundational components of our 
```

Comment:
Yeah basically I try to manually format it by giving line numbers to each line however when I change the size of windows then it breaks the lines as well. I think its because I use word to write the whole document first and then paste it here maybe that is the problem.